### PR TITLE
Bumped latest version to 5.0, updated script to get core contributors, and update core contributor list

### PR DESCRIPTION
**Pull request by astrofrog**
It looks like the getteam.py hadn't been used for a while and was broken but it works again now. To review the changes in the credits page I recommend ignoring whitespace.

### CODE REVIEW

File: getteam.py [hunk 1]
```diff
@@ -96,11 +96,14 @@ def process_html(fn, newcoordinators, newcontributors, indent='\t\t\t'):
...
                     incontrib = False
+                else:
+                    if '<ul class="team">' in l:
+                        lines.append(l)
                 #skip otherwise
             else:
                 if '<ul class="coordinators">' in l:
                     incoord = True
-                elif '<ul class="team">' in l:
+                elif '<h3 id="core-package-contributors">' in l:
```
Comment (astrofrog) [inline getteam.py:106]: This is needed otherwise we update the voting members list

File: getteam.py [hunk 2]
```diff
@@ -115,8 +118,8 @@ def process_html(fn, newcoordinators, newcontributors, indent='\t\t\t'):
     coordinators = extract_names_list(dt, 'Astropy Project Coordinators')
     contributors = extract_names_list(dt, 'Core Package Contributors')
 
-    newhtml = process_html('about.html', coordinators, contributors)
-    print('Replacing "about.html" with updated version.  Be sure to "git diff '
-          'about.html" before committing to ensure no funny  business happened.')
-    with open('about.html', 'w') as f:
+    newhtml = process_html('team.html', coordinators, contributors)
+    print('Replacing "team.html" with updated version.  Be sure to "git diff '
+          'team.html" before committing to ensure no funny  business happened.')
```
Comment (pllim) [inline getteam.py:123]:
```suggestion
          'team.html" before committing to ensure no funny business happened.')
```

**Review by pllim — APPROVED** [review]
If it works for you, good enough for me!

File: team.html [hunk 3]
```diff
@@ -191,403 +191,455 @@ <h2 id="contributors">Contributors<a class="paralink" href="#contributors" title
...
+			<li>Leonardo Ferreira</li>
+			<li>Lia Corrales</li>
+			<li>Lingyi Hu</li>
+			<li>Lisa Martin</li>
+			<li>Lisa Walter</li>
+			<li>Ludwig Schwardt</li>
+			<li>Luigi Paioro</li>
+			<li>Luke G. Bouma</li>
+			<li>Luke Kelley</li>
+			<li>luz paz</li>
```
Comment (eteq) [inline team.html:440]: Isn't this an actual proper name?
```suggestion
			<li>Luz Paz</li>
```
(I know it's derived from the core repo name list, so I can correct it there if you confirm I'm not missing something, @astrofrog )

File: announcements/release-5.0.html [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+		<p>
+		We are very happy to announce the v5.0 release of astropy,
+		a core Python package for Astronomy:
+		</p>
+		<p align='center'>
+		    <img src="astropy_logo_notext.png" style="width:80px;height:80px;"><br>
+		    <a href="http://www.astropy.org">http://www.astropy.org</a>
+		</p>
+		<p>
+		The core astropy package is a community-driven Python package intended to contain much of the
```
Comment (eteq) [inline announcements/release-5.0.html:88]:
```suggestion
		The astropy core package is a community-driven Python package intended to contain much of the
```
?

**Review by eteq — APPROVED** [review]
LGTM aside from a couple of minor suggestions!

File: team.html [hunk 5]
```diff
@@ -191,403 +191,455 @@ <h2 id="contributors">Contributors<a class="paralink" href="#contributors" title
...
-<li>  Yash Kumar </li>
-<li>  Yash Sharma </li>
-<li>  Yingqi Ying </li>
-<li>  Zac Hatfield-Dodds </li>
-<li>  Zach Edwards </li>
-<li>  Zachary Kurtz </li>
-<li>  Zeljko Ivezic </li>
-<li>  Zlatan Vasović </li>
-<li>  Zé Vinicius </li>
+			<li>Aaron Meisner</li>
```
Comment (eteq) [inline team.html:195]:
```suggestion
			<!--- the list below is auto-generated from getteam.py, use that to update --->
			<li>Aaron Meisner</li>
```

File: team.html (hunk 3 above)
Comment (astrofrog) [reply team.html:440]: Sounds good, feel free to add to mailmap

**Pull request merged**